src/pages/course/Course.page.tsx

```diff
@@ -45,22 +42,22 @@ export const Course: FC = () => {
     []
   );
 
-  const handleChangeLessonData = (
-    videoSrc: string,
-    imagePreviewLink: string
-  ): void => {
-    setHlsUrl(videoSrc);
-    setLessonPreviewImgUrl(imagePreviewLink);
-  };
+  const handleChangeLessonData = useCallback(
+    (videoSrc: string, imagePreviewLink: string): void => {
```

Comment:
You have a lot of function that returns nothing. It depends on team decision, so you should choose either write everywhere `void` type or just delete this one :)

Reply:
If function has no parameters and returns void - i do not specifying that. But if any of those are present I specify parameter types and return type of function.